### PR TITLE
docs(VTable): Add description to hover prop

### PR DESCRIPTION
**Pull request by worldpwn**
## Description
Add a description to hover props in the documentation.

### CODE REVIEW

File: packages/api-generator/src/locale/en/v-table.json [hunk 1]
```diff
@@ -3,6 +3,6 @@
     "fixedHeader": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-table.json))",
     "fixedFooter": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-table.json))",
     "height": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-table.json))",
-    "hover": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-table.json))"
+    "hover": "If hover is true it will add a transition effect to a table's row when the mouse is over it."
```
Comment (nekosaur) [inline packages/api-generator/src/locale/en/v-table.json:6]: I think transition is the wrong word to use here. It should simply say we apply a hover effect, or a different background

Comment (worldpwn) [reply packages/api-generator/src/locale/en/v-table.json]: Hi, yes it makes sense. Changed the word in the next commit.

File: packages/api-generator/src/locale/en/v-table.json [hunk 2]
```diff
@@ -3,6 +3,6 @@
     "fixedHeader": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-table.json))",
     "fixedFooter": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-table.json))",
     "height": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-table.json))",
-    "hover": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/master/packages/api-generator/src/locale/en/v-table.json))"
+    "hover": "If hover is true it will add a hover effect to a table's row when the mouse is over it."
```
Comment (nekosaur) [inline packages/api-generator/src/locale/en/v-table.json:6]:
```suggestion
    "hover": "Will add a hover effect to a table's row when the mouse is over it."
```

It sounded a bit repetitive with using hover twice.

**Comment by worldpwn** [comment]
@nekosaur the failing tests are not related to my changes and I cannot rerun them

**Pull request merged**